src/Elcodi/Component/Tax/Tests/UnitTest/Entity/TaxTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Elcodi package.
+ *
+ * Copyright (c) 2014-2015 Elcodi Networks S.L.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ * @author Aldo Chiecchia <zimage@tiscali.it>
+ * @author Elcodi Team <tech@elcodi.com>
+ */
+
+namespace Elcodi\Component\Tax\Tests\UnitTest\Entity;
+
+use Elcodi\Component\Core\Tests\UnitTest\Entity\AbstractEntityTest;
+use Elcodi\Component\Core\Tests\UnitTest\Entity\Traits;
+use Elcodi\Component\Tax\Entity\Tax;
+
+class TaxTest extends AbstractEntityTest
+{
+    use Traits\IdentifiableTrait, Traits\EnabledTrait;
+
+    /**
+     * Sets up the fixture, for example, opens a network connection.
+     * This method is called before a test is executed.
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+        
+        $this->object = new Tax();
+    }
+
+    /**
+     * Return the entity namespace.
+     *
+     * @return string Entity namespace
+     */
+    public function getEntityNamespace()
+    {
+        return 'Elcodi\Component\Tax\Entity\Tax';
+    }
+
+    /**
+     * Return the fields to test in entities.
+     *
+     * [
+     *      [
+     *          "type" => $this::GETTER_SETTER,
+     *          "getter" => "getValue",
+     *          "setter" => "setValue",
+     *          "value" => "Elcodi\Component\...\Interfaces\AnInterface"
+     *          "nullable" => true
+     *      ],
+     *      [
+     *          "type" => $this::ADDER_REMOVER|$this::ADDER_REMOVER,
+     *          "getter" => "getValue",
+     *          "setter" => "setValue",
+     *          "adder" => "addValue",
+     *          "removed" => "removeValue",
+     *          "nullable" => false,
+     *          "bag" => "collection", // can be array
+     *          "value" => "Elcodi\Component\...\Interfaces\AnInterface"
+     *      ]
+     * ]
+     *
+     * @return array Fields
+     */
+    public function getTestableFields()
+    {
+        return [
+            [
+                'type' => $this::GETTER_SETTER,
+                'getter' => 'getName',
+                'setter' => 'setName',
+                'value' => sha1(rand()),
+                'nullable' => false,
+            ],
+            [
+                'type' => $this::GETTER_SETTER,
+                'getter' => 'getDescription',
+                'setter' => 'setDescription',
+                'value' => sha1(rand()),
+                'nullable' => false,
+            ],
+            [
+                'type' => $this::GETTER_SETTER,
+                'getter' => 'getValue',
+                'setter' => 'setValue',
+                'value' => microtime(true),
+                'nullable' => false,
+            ],
+        ];
+    }
+
+    public function testToString()
```

Comment:
We should "standarize" this as well.
Anyway, don't use `$this->object` and create here the mock.